methods.tex

```diff
@@ -160,52 +162,53 @@ \subsection*{\sffamily \large BEM-BEM coupling}
 %
-   \begin{align} \label{eq:pbe_vp}
-\nabla \cdot \left(\epsilon_1(\mathbf{x}) \nabla \phi_1(\mathbf{x})\right) &= \sum_{k=1}^{N_q} q_k\delta(\mathbf{x},\mathbf{x}_k) \quad  \mathbf{x} \in \Omega_1\nonumber\\
-\left(\nabla^2 - \kappa^2\right)\phi_2(\mathbf{x})  &= 0 \quad\mathbf{x}\in\Omega_2\nonumber\\
-\phi_1(\mathbf{x})  = \phi_2(\mathbf{x}),  &\quad \epsilon_1(\mathbf{x})\frac{\partial\phi_1}{\partial\mathbf{n}}(\mathbf{x})  = \epsilon_2(\mathbf{x})\frac{\partial\phi_2}{\partial\mathbf{n}}(\mathbf{x})  \quad \mathbf{x}\in \Gamma. 
+\begin{align} \label{eq:pbe_vp}
+\nabla \cdot \left(\epsilon_1(\mathbf{x}) \nabla \phi_1(\mathbf{x})\right) &= \sum_{k=1}^{N_q} q_k\delta(\mathbf{x},\mathbf{x}_k) &&\mathbf{x} \in \Omega_1,\nonumber\\
+\left(\nabla^2 - \kappa^2\right)\phi_2(\mathbf{x})  &= 0 &&\mathbf{x}\in\Omega_2,\nonumber\\
+\phi_1(\mathbf{x})  &= \phi_2(\mathbf{x}),&&\mathbf{x}\in \Gamma,\nonumber\\
+\epsilon_1(\mathbf{x})\frac{\partial\phi_1}{\partial\mathbf{n}}(\mathbf{x})  &= \epsilon_2\frac{\partial\phi_2}{\partial\mathbf{n}}(\mathbf{x}) &&\mathbf{x}\in \Gamma. 
 \end{align}
 %
 A finite element method (FEM) is more suitable for such a case.
 
-The coupling of finite element (FEM) and boundary element (BEM) methods is a widely used approach to solve multi-physical problems on unbounded domains [citation?], as it takes advantage of both methods. On one hand, the BEM satisfies the infinite boundary conditions exactly when they decay to zero, and only approximates boundary conditions on surfaces, hence, it is commonly used for problems involving infinite or semi-infinite domains. On the other hand, the FEM is known for its robustness and universal applicability, even for problems of inhomogeneous or non-linear nature.
-Here, we used the Johnson-N\'ed\'elec formulation,\cite{johnson1980coupling} which is the simplest form of FEM-BEM coupling, and we detail it next.
+The coupling of FEM and BEM is an approach that can be used to solve a wide range of multiphysics problems on unbounded domains\cite{hiptmair2002,banjai2015}, as it takes advantage of both methods. On the one hand, the BEM satisfies the infinite boundary conditions exactly when they decay to zero, and only approximates boundary conditions on surfaces; hence, it is commonly used for problems involving infinite or semi-infinite domains. On the other hand, the FEM is known for its robustness and universal applicability, even for problems of inhomogeneous or non-linear nature.
```

Comment:
I added a couple of citations of different applications. These may or may not be necessary and may or may not be the best things so cite